std/conv.d

```diff
@@ -3169,21 +2919,29 @@ if (isInputRange!Source && isSomeChar!(ElementType!Source) && !is(Source == enum
             if (i == '.' && !dot)
             {
                 p.popFront();
-                dot++;
+                dot += expIter;
             }
             else
-            {
                 break;
-            }
         }
-        enforce(sawDigits, new ConvException("no digits seen"));
+
+        // Have we seen any mantissa digits so far?
+        enforce(sawDigits, bailOut("no digits seen"));
+        static if (FloatFormat == hex)
```

Comment:
Rebased and made one small tweak, moved this check into `parseDigits()` so it becomes a static check, instead of being done at runtime before.

Reply:
Thought updating might get DAutoTest to work, but looks like it's just broken right now for every pull.